client/src/app/site/motions/components/category-list/category-list.component.html

```diff
@@ -78,28 +79,28 @@ <h2 translate>Categories</h2>
         </mat-expansion-panel-header>
 
         <!-- Edit form shows during the edit event -->
-        <form [formGroup]='this.updateForm' *ngIf="editId === category.id" (keydown)="keyDownFunction($event, category)">
-            <span translate>Edit category details:</span><br>
+        <form id="updateForm" [formGroup]='updateForm' *ngIf="editId === category.id" (keydown)="keyDownFunction($event, category)">
+            <span translate>Edit category details</span>:<br>
             <mat-form-field>
-                <input formControlName="prefix" matInput placeholder="{{'Prefix' | translate}}">
-                <mat-hint>
+                <input formControlName="prefix" matInput placeholder="{{'Prefix' | translate}}" required>
+                <mat-hint *ngIf="!updateForm.controls.prefix.valid">
```

Comment:
should this really only appear when a form is invalid?
This is what we would use MatError for.

https://material.angular.io/components/input/overview
```
<mat-hint>Required</mat-hint>
<mat-error *ngIf="FormControl.hasError('required')">
    This is not optional(or whatever)
</mat-error>
```